### PR TITLE
Standardizing the App Engine readmes. 2/3rds of #137.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/ndb/transactions/README.md [hunk 1]
```diff
@@ -1,61 +1,7 @@
-## NDB Transactions Sample
+## App Engine Datastore NDB Transactions Sample
 
-This is a sample app for Google App Engine that exercises the [NDB Transactions Python API](https://cloud-dot-devsite.googleplex.com/appengine/docs/python/ndb/transactions)
+This is a sample app for Google App Engine that exercises the [NDB Transactions Python API](https://cloud.google.com/appengine/docs/python/ndb/transactions)
```
Comment (waprin) [inline appengine/ndb/transactions/README.md]: lol

File: appengine/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Deploying the samples
+
+1. Download the [Google App Engine Python SDK](https://cloud.google.com/appengine/downloads) for your platform.
+2. Many samples require extra libraries to be installed. If there is a `requirements.txt`, you will need to install the dependencies with [`pip`](pip.readthedocs.org).
+
+        pip install -t lib -r requirements.txt
+
+3. Use `appcfg.py` to deploy the sample, you will need to specify your Project ID and a version number:
+
+        appcfg.py update -A your-app-id -V your-version app.yaml
```
Comment (waprin) [inline appengine/README.md]: I know appcfg is recommended but I feel we could give a shoutout to gcloud as the way forward?

File: appengine/ndb/transactions/README.md (hunk 1 above)
Comment (jonparrott) [inline appengine/ndb/transactions/README.md]: I let out an audible sigh when I saw that.

File: appengine/README.md (hunk 2 above)
Comment (jonparrott) [inline appengine/README.md]: Not for non-managed VMs.

**Comment by waprin** [comment]
LGTM, think we should mention gcloud but don't feel strongly about it.

**Comment by jonparrott** [comment]
Cool, merging.

**Pull request merged**